bolero/representation/dmp_sequence.py

```diff
@@ -323,3 +323,65 @@ def can_step(self):
         """Returns true if step() can be called again, false otherwise."""
         return len(np.where(self.steps <= self.split_steps)[0]) > 0
 
+    def trajectory(self):
```

Comment:
We need unit tests for this function [here](https://github.com/rock-learning/bolero/blob/master/bolero/representation/test/test_dmp_seq.py).

Reply:
Added simple subgoal test